src/metabase/db/custom_migrations.clj

```diff
@@ -1273,12 +1275,16 @@
                                        (= table-name :collection) (assoc :where [:and
                                                                                  [:= :namespace nil] ; excludes the analytics namespace
                                                                                  [:= :personal_owner_id nil]]))]]
-                     [table-name (sort-by :id (map #(into {} %) (t2/query query)))]))]
+                     [table-name (->> (t2/query query)
+                                      (map (fn [x] (into {} (apply dissoc x columns-to-remove))))
+                                      (keep (fn [x] (and (= table-name :collection) (= (:type x) "trash"))))
```

Comment:
We remove the trash collection from the exported data because we don't want to add it again